src/libraries/Native/Unix/System.Globalization.Native/pal_icushim_internal.h

```diff
@@ -55,6 +57,7 @@
 
 #include "pal_compiler.h"
 
+#if !defined(STATIC_ICU)
```

Comment:
Why are we using `STATIC_ICU`? Should we instead just use the `TARGETS_*` defines?

Reply:
it allows to quickly test changes under any other target, not only wasm. e.g. desktop mono

Reply:
Ok, makes sense.